WeakAuras/GenericTrigger.lua

```diff
@@ -3603,7 +3603,7 @@ function GenericTrigger.CreateFallbackState(data, triggernum, state)
   end
 
   if (event.stacksFunc) then
-    local ok, stacks = event.stacksFunc(firstTrigger);
+    local ok, stacks = xpcall(event.stacksFunc, geterrorhandler(), firstTrigger);
```

Comment:
That was broken in ceedae1f0de5a37968ccdd8729a07dfa3f5d692b by me, where I didn't add pcall to it, but to every other call. Somewhat suprisingly that this wasn't noticed, this broke the stacks function in that code path.